dataland-frontend/tests/e2e/utils/SfdrUpload.ts

```diff
@@ -1,11 +1,12 @@
-import { uploadDocuments } from "@sharedUtils/components/UploadDocuments";
 import { TEST_PDF_FILE_NAME } from "@sharedUtils/ConstantsForPdfs";
+import { UploadReports } from "@sharedUtils/components/UploadReports";
 
 /**
  * Adds reports to the dataset via the Sfdr upload form for the given dataset
  */
 export function selectsReportsForUploadInSfdrForm(): void {
-  uploadDocuments.selectFile(TEST_PDF_FILE_NAME, "referencedReports");
```

Comment:
This looks logic-wise very similar to EuTaxonomyNonFinancialsUpload. Would it have merit to further generalize it to reduce code duplications?

Reply:
I see your point. I feel like the tradeoff is readability here. But I have no strong opinion on this. Let me change it.

Reply:
Mislooked. This is already an extracted function and thus can already be reused as is. Thus no arguments against it at all.